src/multiselect.rs

```diff
@@ -422,3 +422,34 @@ impl<'a, T: Display> MultiSelect<'a, T> {
         Ok(())
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use crate::test::without_ansi;
+
+    use super::*;
+    use indoc::indoc;
+
+    #[test]
+    fn test_render() {
+        let select = MultiSelect::new("Toppings")
+            .description("Select your toppings")
+            .option(DemandOption::new("Lettuce").selected(true))
+            .option(DemandOption::new("Tomatoes").selected(true))
+            .option(DemandOption::new("Charm Sauce"))
+            .option(DemandOption::new("Jalapenos").label("Jalapeños"))
+            .option(DemandOption::new("Cheese"))
+            .option(DemandOption::new("Vegan Cheese"))
+            .option(DemandOption::new("Nutella"));
+
+        assert_eq!(
+            indoc! {
+              " Toppings
+             Select your toppings
+           
```

Comment:
no options rendered as paging initialization only happens in `run()`. i assume this might become an issue an a form context where fields should be rendered without a blocking `run()`.